java/WORKSPACE.bazel

```diff
@@ -42,6 +33,14 @@ rules_jvm_external_deps()
 load("@rules_jvm_external//:setup.bzl", "rules_jvm_external_setup")
 rules_jvm_external_setup()
 
+# Load dependencies for the base workspace.
```

Comment:
Can we move the
```
load("@com_google_java_differential_privacy//:dp_java_deps.bzl", "dp_java_deps")
dp_java_deps()
```
Also just before this line?  This way, we would follow the instructions as in https://github.com/bazelbuild/rules_jvm_external/releases/tag/4.4.2.